README.md

```diff
@@ -11,13 +11,21 @@ Here is a mock up of the future design:
 ![Alt text](design/images/TuneUp_Mockup_03_SortedByExecutionOrder.jpg?raw=true "TuneUp")
 
 ## Building
-Recommended build environment
+### Recommended Build Environment
 - VisualStudio 2019
 - .Net Framework 4.7 Developer Pack
 - Dynamo repository cloned and built on the same level of TuneUp repository which means your Dynamo repo and TuneUp repo should exist under the same parent folder.
 
+### Result Binaries
+- Everytime 3rd party dev build TuneUp in `Debug` config, they should expect:
+    - Under `TuneUp\dist\TuneUp`, there is a sample package wrapped up ready for publishment and adoption. This would be the un-optimized version.
+    - Un-optimized package installed locally for [DynamoVersion] defined in TuneUp/TuneUp.csproj, under DynamoCore and DynamoRevit
+- Everytime 3rd party dev build TuneUp in `Release` config, they should expect:
+Under `TuneUp\dist\TuneUp`, there is a sample package wrapped up ready for publishment and adoption. This would be the optimized version.
+
 ## Known issues
 - TuneUp does not work with .dyfs (custom nodes) yet.
+- TuneUp does not work in a sementic versioning way, so APIs are not guaranteed to be backward compatible
```

Comment:
hmm, maybe - TuneUp binaries are not semantically versioned and are not intended to be built on top of as an API. Do not treat these binaries like DynamoCore?